script/test

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# script/test: Run the test suite for the application. Optionally pass in a path
+#              to an individual test file to run a single test.
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+if [ -n "$DEBUG" ]; then
+  set -x
+fi
+
+set +e
+echo "==> Setting up test user..."
+createuser --super test >/dev/null 2>&1
+set -e
+
+TEST_FILE=$1
+
+if [ -n "$TEST_FILE" ]; then
+  echo "==> Running the tests matching '$TEST_FILE'..."
+  bundle exec rspec --pattern "$TEST_FILE"
+else
+  if [ -n "$CI" ]; then
+    echo "==> Linting Ruby..."
+    bundle exec standardrb
+  else
+    echo "==> Linting Ruby in fix mode..."
+    bundle exec standardrb --fix
```

Comment:
This is nice! :+1:

Reply:
I nicked it off the Rails template, so can't claim credit 😄